spec/mac_app_store_dumper_spec.rb

```diff
@@ -59,6 +59,18 @@
         409201541 Pages (7.1)
         123456789 123AppNameWithNumbers (1.0)
         409203825 Numbers (5.1)
+        944924917 Pastebin It! (1.0)
+        123456789 My (cool) app (1.0)
+        987654321 an-app-i-use (2.1)
+        123457867 App name with many spaces (1.0)
+        893489734 my,comma,app (2.2)
+        832423434 another_app_name (1.0)
+        543213432 My App? (1.0)
+        688963445 app;with;semicolons (1.0)
+        123345384 my 😊 app (2.0)
+        896732467 你好 (1.1)
+        634324555 مرحبا (1.0)
```

Comment:
this looks different to what I have locally, maybe something for @MikeMcQuaid to investigate internally 😄